updates to kyrgyz language

JRMeyer opened the pull request:
mostly the ky_list file has been updated

rhdunn commented:
Thanks for the PR. I've removed the `#phonemes#` file as that is not present in eSpeak NG and updated the ph_kyrgyz file to use the current name changes. You can proceed to work from the latest master branch to make it easier to merge future changes.

JRMeyer commented:
Thank you for maintaining such a great open source software like this! The
fact that it gets used as the engine in NVDA makes a real difference in the
lives of people who are blind:)

I'll work from the master branch from now on.

I submitted an issue I'm having with reading digits (numbers)... If you
have some pointers I'd greatly appreciate it.

Best,
Josh




вторник, 14 февраля 2017 г. пользователь Reece H. Dunn написал:

> Thanks for the PR. I've removed the #phonemes# file as that is not
> present in eSpeak NG and updated the ph_kyrgyz file to use the current name
> changes. You can proceed to work from the latest master branch to make it
> easier to merge future changes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/espeak-ng/espeak-ng/pull/224#issuecomment-279633860>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIAE6Me1slGVsrA7YNX6vrYLqRSf1DM2ks5rcV9PgaJpZM4MAKD_>
> .
>


-- 
Joshua Meyer
Ph.D. Student
Linguistics Department
University of Arizona
jrmeyer.github.io
